_shared/project/tests/functional/pytest_plugins/db.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from tests.factories.factoryboy_sqlalchemy_session import (
+    clear_factoryboy_sqlalchemy_session,
+    set_factoryboy_sqlalchemy_session,
+)
+
+
+@pytest.fixture
+def db_session(db_engine, db_sessionfactory):  # pragma: no cover
+    """Return a standalone database session for preparing database state."""
+
+    connection = db_engine.connect()
+    session = db_sessionfactory(bind=connection)
+    set_factoryboy_sqlalchemy_session(session)
+
+    try:
+        yield session
+    finally:
+        clear_factoryboy_sqlalchemy_session()
+        session.close()
+        connection.close()
```

Comment:
`db_session` fixture for functests. Projects can override this fixture in their `conftest.py`'s or add this entire file to their ignore lists.